manim/scene/vector_space_scene.py

```diff
@@ -759,22 +759,22 @@ def add_transformable_label(
         vector : Vector
             The vector for which the label must be added.
 
-        label : TexMobject,str
-            The TexMobject/string of the label.
+        label : MathTex,str
```

Comment:
let's already start enforcing the guidelines here
```suggestion
        label : Union[:class:`~.MathTex`, :class:`str`]
```